docs/deployment/cognito-rds-s3/README.md

```diff
@@ -0,0 +1,111 @@
+# Deploying Kubeflow with Amazon Cognito as idP, RDS and S3
+
+This guide describes how to deploy Kubeflow on AWS EKS using Cognito as identity provider, RDS for database and S3 for artifact storage.
+
+## 1. Prerequisites
+Follow the pre-requisites section from [this guide](../rds-s3/README.md#1-prerequisites) to:
+1. Install the CLI tools
+1. Clone the repo
+1. Create an EKS cluster and
+1. Create S3 Bucket
+1. Create RDS Instance
+1. Configure AWS Secrets for RDS and S3
+1. Install AWS Secrets and Kubernetes Secrets Store CSI driver
+
+## Configure RDS endpoint and S3 bucket name for Kubeflow Pipelines
+
+Follow the [Configure Kubeflow Pipelines](../rds-s3/README.md#2-configure-kubeflow-pipelines) section from this guide to:
+1. Substitute the RDS Host and S3 Bucket name used to access the dbinstance and S3
+
+## Configure Custom Domain and Cognito
+
+1. Follow the [cognito guide](../cognito/README.md) from [section 1.0(Custom Domain)](../cognito/README.md#10-custom-domain) upto [section 4.0(Configure Ingress)](../cognito/README.md#40-configure-ingress) to:
+    1. Create a custom domain
+    1. Create TLS certificates for the domain
+    1. Create a Cognito Userpool
+    1. Configure Ingress
+2. Deploy Kubeflow. Choose one of the two options to deploy kubeflow:
+    1. **[Option 1]** Install with a single command
+        ```
+        while ! kustomize build docs/deployment/cognito-rds-s3 | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+        ```
+    1. **[Option 2]** Install individual components
```

Comment:
In future might be it good to call out which of these individual components are necessary for installation?